File: app/src/main/java/com/example/pluu/myapplication/MainActivity.java
package com.example.pluu.myapplication;

import android.graphics.Typeface;
import android.support.v4.content.res.ResourcesCompat;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        TextView tv = findViewById(android.R.id.text1);
        Typeface font = ResourcesCompat.getFont(this, R.font.arizonia);
        tv.setTypeface(font);
    }
}

File: app/src/main/java/com/example/pluu/myapplication/MainActivity.kt
package com.example.pluu.myapplication

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
//        findViewById<TextView>(android.R.id.text1)?.apply {
//            val font: Typeface? = ResourcesCompat.getFont(context, R.font.arizonia)
//            typeface = font
//        }

//        val request = FontRequest("com.example.fontprovider.authority",
//                "com.example.fontprovider", "my font", certs)
//        val callback = FontsContractCompat .FontRequestCallback()
//
//        FontsContractCompat.requestFont(this, request, callback, Handler())

    }
}
